apps/web/hooks/useBucket.tsx

```diff
@@ -35,6 +35,8 @@ export default function useBucket(): ContextValue {
 		return useS3();
 	} else if ((Provider[keys.type] as Provider) === Provider.wasabi) {
 		return useS3();
+	} else if ((Provider[keys.type] as Provider) === Provider.digitalocean) {
+		return useS3();
```

Comment:
Shall we consider turning this into a switch statement?
https://github.com/nghiapham1026/firefiles/blob/3650bf44eaf6fc21eb2acb5062d6bc277870f180/apps/web/hooks/useBucket.tsx#L28-L40
Something along the lines of:
```
// store the type of key as a variable to avoid repetition
providerKey = (Provider[keys.type] as Provider;

// view this example https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch#using_switch
switch (providerKey):
   case Provider.s3:
   case Provider.backblaze:
   case Provider.wasabi:
   case Provider.digitalocean:
      return useS3();
   case Provider.firebase:
      return useFirebase();
   case default:
      return null;
```